index.html

```diff
@@ -310,9 +310,7 @@ <h3>
           The <dfn>request()</dfn> method
         </h3>
         <p data-tests="wakelock-type.https.any.html">
-          The {{WakeLock/request()}} method, when invoked, MUST run the
-          following steps. The method takes one argument, the {{WakeLockType}}
-          |type:WakeLockType|:
+          The <code>request(|type:WakeLockType|)</code> method steps are:
```

Comment:
@marcoscaceres I'm not sure if this is the right change, or if there's a way to make ReSpec use the previous notation while also defining `type`'s type.

Reply:
@marcoscaceres friendly ping

Reply:
Oops! Sorry, will review tomorrow 🙏

Reply:
@marcoscaceres gentle reminder :-)

Reply:
heh, 21 days ago... looking now...